server/tests/FullTest.cc

```diff
@@ -0,0 +1,95 @@
+#include "FullTest.hh"
+#include "dto/PlaceDto.hh"
+
+#define CONFIG_ALLOW_MUTATION
```

Comment:
why is there a macro redefinition?

Reply:
This macro should be defined in this place only, but MSVC throws linking errors, when I don't enable it globaly.

It is defined globaly only, when compiling with MSVC, and redefinition here is harmless.